coderd/healthcheck/healthcheck.go

```diff
@@ -136,6 +145,18 @@ func Run(ctx context.Context, opts *ReportOptions) *Report {
 		report.Database = opts.Checker.Database(ctx, &opts.Database)
 	}()
 
+	wg.Add(1)
```

Comment:
if we know that there is only one function, do we need the wait group? Is it because of `recover()`?

Reply:
Honestly, not sure. I was just copy-pasta-ing what was already there.

Reply:
Yep, as you suspected, we need to run the healthcheck report in a goroutine so we can `recover()`, and also so we can do them all in parallel. We need the waitgroup to synchronize all of those goroutines.

Reply:
Why do we panic in these functions vs returning the value/error? Or is this just a precaution for unforeseen situations?

Reply:
I think it's more the latter. Healthcheck funcs ideally shouldn't panic.